pom.xml

```diff
@@ -92,8 +92,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <source>11</source>
-          <target>11</target>
+          <source>14</source>
```

Comment:
Jak chcecie zmienić JDK z wersji 11 do wersji 14 to musicie chyba też zmienić linię 17 i 18 w pliku pom.xml

Reply:
ja bym chyba została przy 11, żeby reszcie przypadkiem nie zamieszać, bo chyba zdecydowana większość pracuje na 11 :/

Reply:
spoko, wróciłem do wersji 11, ale myślę, że prędzej czy później problem wróci :P